### PR TITLE
Flexibus/#45/user dashboard

### LINKED ISSUE #45: User-Dashboard
AS a user after successful log-in
I WANT to come to User-Dashboard page
TO see the most important information about upcoming events

- We DO NOT implement tables or any content of Dashboard page yet.
- Successful login => redirect user to his dashboard (for example .../my)
- There is link "My Dashboard" in User Dropdown Menu

### PR DESCRIPTION
**Pull request by deeps96**
Closes #45.

Adding a link to the users dashboard, created by #45, to the drop down menu #37.

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -41,6 +41,10 @@ de:
     index:
       lead: "Wir bieten unglaubliche Lösungen für vielfältige Use Cases."
 
+  user-dashboard:
```
Comment (jkbe) [inline config/locales/de.yml]: Call this navbar maybe? Hard to say whether it belongs to navbar or dashboard, just a proposal.

File: config/locales/de.yml [hunk 2]
```diff
@@ -41,6 +41,10 @@ de:
...
+    drop-down:
+      dashboard: "My Dashboard"
```
Comment (jkbe) [inline config/locales/de.yml]: German please.

**Review by jkbe — CHANGES_REQUESTED** [review]

File: config/locales/de.yml (hunk 2 above)
Comment (deeps96) [reply config/locales/de.yml]: Is it ok to change it to "Mein Dashboard"?
"Instrumententafel" and "Armaturenbrett" sound weird in this context.

Comment (jkbe) [reply config/locales/de.yml]: Haha sure.

Comment (jkbe) [reply config/locales/de.yml]: @zhamanakov ?

**Review by jkbe — APPROVED** [review]

**Pull request merged**